Implemented the superping feature.

FloareDor opened the pull request:
### Added the superping command to the Bot.
* The Bot can take input for superpinging multiple users, at once.

FloareDor committed:
Init random cogs

FloareDor committed:
Added the `superping` feature

AnantaSrikar reviewed (changes requested):
Few changes will be needed:

1. Rename the class from `random` to `admin_misc` in `cog/admin/admin_misc.py`.
2. Edge cases have to be handled for `superping`.
3. Removed unused imports.
4. Move the `purge` based commands to `cogs/admin/mod.py`.

FloareDor committed:
Handled more edge cases for the `superping` command.

FloareDor commented:
#### Changes made:
* Renamed the `random` class to `admin_misc` in `cog/admin/admin_misc.py`.
* Made changes so that the bot handles all the edge cases known for `superping`. 
* Removed unecessary imports.
* Relocated the purge commands from `cogs/admin_misc.py` to `cogs/mod.py`.

AnantaSrikar reviewed (approved):
Good work! Will merge.
